apps/suite-launch/test-presets/mountebank-configurable/fluent-bit.conf

```diff
@@ -0,0 +1,95 @@
+[SERVICE]
+     # See:
+     # https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html
+     # https://github.com/aws-samples/amazon-ecs-firelens-under-the-hood/tree/master/generated-configs/fluent-bit
+     Flush        10
```

Comment:
Why such a high value? For high throughput tests this is not ideal I think

Reply:
@matthewfala ?

Reply:
Shoots, I missed this one. Let me change it.